feature/interface/staticarp/ate_tests/static_arp_test/static_arp_test.go

```diff
@@ -153,11 +153,12 @@ func configureDUT(t *testing.T, peermac string) {
 	i1 := &telemetry.Interface{Name: ygot.String(p1.Name())}
 	d.Interface(p1.Name()).Replace(t,
 		configInterfaceDUT(i1, &dutSrc, &ateSrc, peermac))
-
+	dutSrc.AddToDUT(t, d, i1, 0)
```

Comment:
I was thinking that attrs.AddToDUT should just do the d.Interface().Replace() as well. It will take p1 as argument instead of i1. It should not need to take the si argument since it's okay to assume it will be 0.

Reply:
Sounds good! I'll make those changes and commit once verified/tested locally.

Reply:
Couple of comments here:

1. `AddToDUT` seems a strange name -- since it's not very clear what is actually happening -- can this be called `AssignPortsToNI`? 
2. @liulk - from a featureprofiles perspective, is the plan to transition everything to use attrs per @Hobbydos' question? It seems like this might need to be done before having a common pattern here can be implemented. Is there a LSC that we're already doing to implement that?

Reply:
Name `AddToDUT` is chosen as an analogy to an existing method `AddToATE`. Perhaps not the best name, but I want to highlight the similarity of their purpose. We can change both to `SetOnDUT` and `SetOnATE` but that requires a LSC. I'm also okay to rename `AddToATE` to `SetOnATE` provided that a compatibility function `AddToATE` is retained and marked as deprecated so we remember to clean it up.

Reply:
1. Ah, so this function is intended to do more than solely this  -- OK, thanks. I had not understood this. @Hobbydos please can we update the doc comment so that it doesn't specifically say it is going to do this.
2. Thanks -- so if we're going through these helpers, that SGTM. @Hobbydos - how many places did you find that need alternate implementations?

Reply:
@robshakir; at least 25 - here's a quickly generated list.

```
grep -r "configInterfaceDUT(" | awk '{print $1}' | cut -d ":" -f1 | uniq
feature/experimental/gribi/ate_tests/dut_daemon_failure/dut_daemon_failure_test.go
feature/experimental/p4rt/ate_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go
feature/experimental/p4rt/ate_tests/google_discovery_protocol_packetout_test/google_discovery_protocol_packetout_test.go
feature/experimental/policy/policy_vrf_selection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go
feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go
feature/experimental/policy/policy_vrf_selection/otg_tests/base_vrf_selection/base_vrf_selection_test.go
feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go
feature/gribi/ate_tests/ack_in_presence_other_routes/route_ack_test.go
feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go
feature/gribi/ate_tests/base_leader_election_test/base_leader_election_test.go
feature/gribi/ate_tests/get_rpc_test/get_rpc_test.go
feature/gribi/ate_tests/hierarchical_weight_resolution_test/hierarchical_weight_resolution_test.go
feature/gribi/ate_tests/leader_failover_test/leader_failover_test.go
feature/gribi/ate_tests/ordering_ack_test/ordering_ack_test.go
feature/gribi/ate_tests/supervisor_failure_test/supervisor_failure_test.go
feature/gribi/otg_tests/ack_in_presence_other_routes/route_ack_test.go
feature/gribi/otg_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go
feature/gribi/otg_tests/base_leader_election_test/base_leader_election_test.go
feature/gribi/otg_tests/get_rpc_test/get_rpc_test.go
feature/gribi/otg_tests/leader_failover_test/leader_failover_test.go
feature/gribi/otg_tests/ordering_ack_test/ordering_ack_test.go
feature/interface/singleton/ate_tests/singleton_test/singleton_test.go
feature/interface/singleton/otg_tests/singleton_test/singleton_test.go
feature/interface/staticarp/ate_tests/static_arp_test/static_arp_test.go
feature/interface/staticarp/otg_tests/static_arp_test/static_arp_test.go
```